apps/ui/src/components/checkbox/styles/base.css

```diff
@@ -0,0 +1,26 @@
+.input {
+  /* input reset */
+  appearance: none;
+  -webkit-appearance: none;
+  -moz-appearance: none;
```

Comment:
I recommend removing `vendor prefixies`  because the PostCSS - autoprefixes added them automatically